src/components/jogging/JoggingCartesianAxisControl.tsx

```diff
@@ -35,54 +38,81 @@ export const JoggingCartesianAxisControl = externalizeComponent(
 
         element.textContent = displayValue
       })
+      const theme = useTheme()
+      const [borderColor, setBorderColor] = useState(colors?.borderColor)
 
       const valueContainerRef = useRef<HTMLParagraphElement>(null)
 
-      function onPointerDownMinus(ev: React.PointerEvent) {
-        if (disabled) return
-
-        // Stop right click from triggering jog
-        if (ev.button === 0) startJogging("-")
-      }
-
-      function onPointerDownPlus(ev: React.PointerEvent) {
-        if (disabled) return
-
-        if (ev.button === 0) startJogging("+")
-      }
-
       if (!colors) {
         colors = {
           color: "#fff",
           backgroundColor: "#000",
           borderColor: "#000",
-          buttonBackgroundColor: "#000",
+          buttonBackgroundColor: {
+            disabled: "#000",
+            default: "#000",
+            hovered: "#000",
+            pressed: "#000",
+          },
         }
       }
 
+      const SxAxisControlButton = {
+        width: "55px",
+        backgroundColor: colors.buttonBackgroundColor?.default,
+        color: colors.color,
+        alignContent: "center",
+        fontSize: "37px",
+        "&:hover": {
+          backgroundColor: colors.buttonBackgroundColor?.hovered,
+        },
+        "&:active": {
+          backgroundColor: colors.buttonBackgroundColor?.pressed,
+        },
+        ":disabled": {
+          backgroundColor: colors.buttonBackgroundColor?.disabled,
+          "svg path": { fill: theme.palette.action.disabled },
+        },
+      }
+
+      function onPointerDown(ev: React.PointerEvent, direction: Direction) {
+        if (disabled) {
+          return
+        }
+        setBorderColor(colors?.buttonBackgroundColor?.pressed)
+        if (ev.button === 0) {
+          startJogging(direction)
+        }
+      }
+
+      function onPointerUp() {
```

Comment:
I'd probably rename this to onPointerUpOrOut and put the stopJogging call inside it

Reply:
Done